hardware/areca.py

```diff
@@ -26,51 +26,52 @@
 
 
 def split_units(lis):
-    'If the value have unit, remove it from the value and return the unit name'
+    # If the value have unit, remove it from the value
+    # and return the unit name
     for unit in ['RPM', '%', ' V', ' C', 'Seconds', 'Times', 'MHz', 'KB', 'MB',
                  'GB']:
-        match = re.search("(.*)%s$" % unit, lis[1])
```

Comment:
All these changes do not look super useful to me.. but okay

Reply:
The string wrap change from " to ' was almost entirely automated, I can drop it entirely if you think it's not useful at all.
Also I understand it's a pain to review.
The idea was to start having some consistency between modules, since in some places we use " and others '.